lib/api/objectPut.js

```diff
@@ -10,14 +10,14 @@ import { logger } from '../utilities/logger';
 function _storeInMDandDeleteData(bucketName, objMD, dataGetInfo,
     metadataStoreParams, dataToDelete, deleteLog, callback) {
     services.metadataStoreObject(bucketName, objMD, dataGetInfo,
-        metadataStoreParams, (err, contentMD5) => {
+        metadataStoreParams, (err, value) => {
```

Comment:
This is another change in the S3-metadata communication. Now metadata is sending back `ETag` to S3. In the case of versioning, it needs to include the `versionId` as well.